maintainers/maintainer-list.nix

```diff
@@ -16073,4 +16073,10 @@
     github = "ziguana";
     githubId = 45833444;
   };
+  tehmatt = {
```

Comment:
Please, add yourself to maintainer-list as separate commit.

Reply:
Done